acceptance/ui/features/steps/hosts_steps.rb

```diff
@@ -31,6 +72,27 @@ def visitHostsPage()
     expect(@hosts_page).to be_displayed
 end
 
-def clickAddHostsButton()
-    @hosts_page.addHosts_button.click
+def clickAddHostButton()
+    @hosts_page.addHost_button.click
+end
+
+def fillInDefaultHostAndPort()
+    @hosts_page.hostName_input.set "172.17.42.1:4979"
 end
+
+def fillInDefaultResourcePool()
+    @hosts_page.resourcePool_input.select "default"
+end
+
+def fillInDefaultRAMCommitment()
+    @hosts_page.ramCommitment_input.set "50%"
+end
+
+def removeAllHosts()
+    Capybara.match=:first
+    while page.all("[ng-repeat='host in $data']").size != 0 do
+      click_link_or_button("Delete")
+      click_link_or_button("Remove Host")
+    end
+    Capybara.match=:smart
```

Comment:
A downside of using third-party libraries is getting burned when their behavior changes with a new release. So we want to insulate ourselves as much as possible from assumptions that might change. In this case it would be safer to not assume the default value of Capybara.match is :smart. We can do that by saving the current value before we do anything, and restoring the saved value when we're done:

```
    defaultMatch = Capybara.match
    Capybara.match = :first
    while page.all("[ng-repeat='host in $data']").size != 0 do
      click_link_or_button("Delete")
      click_link_or_button("Remove Host")
    end
    Capybara.match = defaultMatch
```